submissions/yaripey/memory-pair-game/js/main.js

```diff
@@ -0,0 +1,10 @@
+import { warning, greetings } from './scenes.js'
+import { Game } from './game.js'
+
+// Starting scenes before the main game starts
+const script = [warning, greetings]
+
+var game = new Game();
+game.startGame(script)
```

Comment:
`var`?
Why not `const game = new Game([warning, greetings])` with "autostart" (i.e. calling `this.startGame()` in constructor)?

Besides `Game` part in `game.startGame` looks redundant